CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ This file follows [Keepachangelog](https://keepachangelog.com/) format.
 Please add your entries according to this format.
 
 ## Unreleased
+* added scroll to highlighted text search in response screen
```

Comment:
Can you move this to Added?

Also:
```suggestion
* Added scroll to highlighted text search in response screen [#988]
```

and add a link a the bottom of the file

Reply:
Sure